api_products.md

```diff
@@ -125,19 +125,32 @@ Weighing attributes:
 | Parameter         | Type       | Default | Description                                                                           |
```

Comment:
Zeile 112: `there are three cases` stimmt so nicht mehr bzw man sollte die Fälle verallgemeinern. 
`The code contains the weight in grams` -> wäre dann eher sowas wie `The code contains the weight in one of the supported units` oÄ